lib/build-manifest-writer.js

```diff
@@ -12,8 +12,12 @@ module.exports = manifestConfig => {
 	let manifest = {};
 	return results => {
 		let filteredResults = results.filter(result => result.changed);
-		manifest = updateManifest(filteredResults, manifest, manifestConfig);
-		return writeManifest(filteredResults, manifest, manifestConfig);
+		if(filteredResults.length > 0) {
```

Comment:
FWIW, this is an example where I'd prefer early return:

```javascript
if(!filteredResults.length) {
    return [];
}

manifest = updateManifest(filteredResults, manifest, manifestConfig);
return writeManifest(filteredResults, manifest, manifestConfig);
```

Reply:
Addressed in https://github.com/faucet-pipeline/faucet-pipeline-static/pull/6